File: QuanLyPhongKham.Client/src/interfaces/responses/PhieuKhamResponse.js
/**
 * @typedef {Object} PhieuKhamResponse
 * @property {string} maPK
 * @property {string|Date} ngayKham
 * @property {string} trieuChung
 * @property {string} chuanDoan
 * @property {string} huongDieuTri
 * @property {string} trangThaiTiepNhan
 * @property {string} maLH
 * @property {string} maBS
 * @property {string} tenBacSi
 * @property {string} tenBenhNhan
 */

export const createPhieuKhamResponse = (
  maPK = "",
  ngayKham = null,
  trieuChung = "",
  chuanDoan = "",
  huongDieuTri = "",
  trangThaiTiepNhan = "",
  maLH = null,
  maBS = null,
  tenBacSi = "",
  tenBenhNhan = "",
) => ({
  maPK,
  ngayKham,
  trieuChung,
  chuanDoan,
  huongDieuTri,
  trangThaiTiepNhan,
  maLH,
  maBS,
  tenBacSi,
  tenBenhNhan,
});

File: QuanLyPhongKham.Server/src/QuanLyPhongKham.DTOs/Responses/PhieuKhamResponse.cs
namespace QuanLyPhongKham.DTOs.Responses
{
    public class PhieuKhamResponse
    {
        public Guid MaPK { get; set; }
        public DateTime NgayKham { get; set; }
        public string? TrieuChung { get; set; }
        public string? ChuanDoan { get; set; }
        public string? HuongDieuTri { get; set; }
        public string? TrangThaiTiepNhan { get; set; }
        public Guid? MaLH { get; set; }
        public Guid? MaBS { get; set; }
        public string? TenBacSi { get; set; }
        public string? TenBenhNhan { get; set; }
    }
}
